convert/src/label_studio_to_badgerdoc/badgerdoc_format/pdf_renderer.py

```diff
@@ -46,7 +50,7 @@ def _draw_token(self, token: BadgerdocToken, page: Page) -> None:
         rect = fitz.Rect(token.bbox)
         shape = page.new_shape()
         shape.draw_rect(rect)
-        shape.insert_textbox(
-            rect, token.text, fontname=self.font_name, fontsize=self.font_size
+        shape.insert_text(
+            token.bbox[:2], token.text, fontname="cjk", fontsize=self.font_size
```

Comment:
again, we might better pass font name once somewhere while initializing class, but not like hardcoded param.